solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1215,16 +1210,16 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
     /**
      * @dev returns the average number of decimal digits in a given list of positive integers
      *
-     * @param _values  list of positive integers
+     * @param list list of positive integers
      *
      * @return the average number of decimal digits in the given list of positive integers
      */
-    function geometricMean(uint256[2] memory _reserveAmounts) private pure returns (uint256) {
-        uint256[] memory reserveAmounts = new uint256[](2);
+    function geometricMean(uint256[2] memory list) private pure returns (uint256) {
+        uint256[] memory dynamicArr = new uint256[](2);
```

Comment:
`dynamicArr` isn't a good name. We should always try to use semantically meaningful names

Reply:
Fixed.